protocol/src/main/java/io/camunda/zeebe/protocol/record/intent/Intent.java

```diff
@@ -200,6 +203,8 @@ static Intent fromProtocolValue(final ValueType valueType, final String intent)
         return FormIntent.valueOf(intent);
       case USER_TASK:
         return UserTaskIntent.valueOf(intent);
+      case PROCESS_INSTANCE_MODIFICATION:
```

Comment:
❌ This should be `PROCESS_INSTANCE_MIGRATION` and its intent. 👍

Reply:
Well spotted!